components/contact/ContactProvinceRow.js

```diff
@@ -16,9 +16,9 @@ const ContactProvinceRow = ({ label, items, id }) => {
         >
           {items.map((x, i) => (
             <div className="col-span-2 md:col-span-1 py-3" key={i}>
-              <strong className="prose prose-strong:text-xl prose-strong:font-display prose-p:text-xl prose-p:font-display">
+              <div className="prose prose-strong:text-xl prose-strong:font-display prose-p:text-xl prose-p:font-display">
                 <Markdown>{`${ap(x.content, ' ')}`}</Markdown>
-              </strong>
+              </div>
```

Comment:
While changing this to a div is fine, it would require more work on the AEM side, else we lose the styling which I don't think we want given we want to apply importance to the bold content in this case. 

Rather than change it, we can instead force `markdown-to-jsx` to parse the content as inline (making it a span), which would be fine since we then have an inline element inside of another inline element instead of a block element inside of an inline element.
```suggestion
              <strong className="text-xl font-display">
                <Markdown options={{ forceInline: true }}>{`${ap(
                  x.content,
                  ' '
                )}`}</Markdown>
              </strong>
```